<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>665</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>696</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>view</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="cols_spinbox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>70</y>
      <width>45</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro</family>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="cols_label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Columns</string>
    </property>
   </widget>
   <widget class="QLabel" name="rows_label">
    <property name="geometry">
     <rect>
      <x>557</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro</family>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Rows</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="rows_spinbox">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>100</y>
      <width>45</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro</family>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="open_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>125</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>125</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create Maze</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>125</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gotham Pro</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clear Area</string>
    </property>
   </widget>
   <widget class="QLabel" name="filename_label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="savetofile_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>125</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save to file</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
